### PR TITLE
Magento 2260 magento api endpoint to create

### PR DESCRIPTION
**Pull request by awesselburg**
## Description

Endpoint for customer creation via the Admin token

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

**Comment by augsteyer** [comment]
tests failing, they will need checkin'

File: src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Rest/Admin/V2.php [hunk 1]
```diff
@@ -45,4 +45,71 @@ protected function _update(array $filteredData)
...
+        return $this->filterOutData($customer->getData());
+    }
+
+    /**
+     * @param Varien_Object $requestData
+     *
+     * @return Mage_Core_Model_Store
+     * @throws Mage_Api2_Exception
+     */
+    protected function getStore($requestData)
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Rest/Admin/V2.php]: just noticed something, store_id is not even a required parameter to pass. The only thing the customer needs is a website_id. See https://devdocs.magento.com/guides/m1x/api/rest/Resources/resource_customers.html#RESTAPI-Resource-Customers-HTTPMethod-POST-customers

File: src/app/code/community/Shopgate/Cloudapi/etc/api2.xml [hunk 2]
```diff
@@ -238,6 +241,14 @@
                     <group_id>Group</group_id>
                     <confirmation>Is Confirmed</confirmation>
                     <disable_auto_group_change>Disable automatic group change based on VAT ID</disable_auto_group_change>
+                    <dob>Customer date of birth</dob>
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Cloudapi/etc/api2.xml]: not sure it's needed as it's not in the `app/code/core/Mage/Customer/etc/api2.xml`, but who knows, I will double check when logic is adjusted

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+- Endpoint for customer creation       (POST shopgate/v2/customers)
```
[the remark this answers is not in the document]
Comment (andre-kraus) [reply CHANGELOG.md]: + You forgot the section. I would suggess ### Added

**Comment by augsteyer** [comment]
Swagger done, on you @andre-kraus

**Pull request merged**